osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/services/DeviceManagementService.java

```diff
@@ -469,6 +347,198 @@ public Organisation findOrganisation(@Identification final String organisationId
         return devices;
     }
 
+    private Specifications<Device> doApplyFilter(final DeviceFilter deviceFilter, final Organisation organisation)
+            throws FunctionalException, ArgumentNullOrEmptyException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)